src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+html {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
`html` doesn't have default margins and paddings, so there's no need to reset them.